networks/mainnet.json

```diff
@@ -58,9 +62,10 @@
     "cWBTC": "0xC11b1268C1A384e55C48c2391d8d480264A3A7F4",
     "USDC": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
     "PriceData": "0xc629c26dced4277419cde234012f8160a0278a79",
-    "PriceFeed": "0x922018674c12a7f0d394ebeef9b58f186cde13c1",
+    "PriceFeed": "0x02557a5e05defeffd4cae6d83ea3d173b272c904",
```

Comment:
should this be 0x4007b71e01424b2314c020fb0344b03a7c499e1a instead?

From the proposal: https://compound.finance/governance/proposals/45

Reply:
wth must have rebased away